nilearn/plotting/img_plotting.py

```diff
@@ -12,7 +12,6 @@
 import functools
 import numbers
 import warnings
-
```

Comment:
Not such a big deal for this one but please refrain from making changes like this in the future. It just adds unnecessary noise.